### ISSUE TITLE
ankiconnect  IP地址

### ISSUE THREAD

**Issue by ledvip** [ninja33/ODH#143]
老黄你好！Ankiweb最近一直不能使用。Ankiconnect能否增加一个写ip地址的框，这样便于联到局域网内其它机器的Anki 一般是192.168.X.X:8765

**Comment by ninja33** [comment]
ankiconnect 你可以直接改插件的那个python文件啊。在插件目录的init.py里




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"ledvip"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月19日(星期四) 晚上9:58
收件人:&nbsp;"ninja33/ODH"<ODH@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[ninja33/ODH] ankiconnect  IP地址 (#143)





 
老黄你好！Ankiweb最近一直不能使用。Ankiconnect能否增加一个写ip地址的框，这样便于联到局域网内其它机器的Anki 一般是192.168.X.X:8765

**Comment by ledvip** [comment]
是的。这个在线词典助手的数据输出界面如果能编辑修改默认的ankiconnect的ip127.0.0.1:8765就太好了，这样可以连到局域网或者远程的anki上。
